antenna/breakpad_resource.py

```diff
@@ -376,10 +379,6 @@ def get_throttle_result(self, raw_crash):
         # throttle the crash.
         result, rule_name, throttle_rate = self.throttler.throttle(raw_crash)
 
-        # Save the results in the raw_crash itself
-        raw_crash["legacy_processing"] = result
-        raw_crash["throttle_rate"] = throttle_rate
-
         return result, rule_name, throttle_rate
```

Comment:
This method doesn't do much other than call another thing. I thought about removing this interstitial method, but then decided to think about that later.